.github/workflows/release.yaml

```diff
@@ -54,3 +54,34 @@ jobs:
 
       - name: Run actual integration test
         run: bash connaisseur/tests/integration/integration-test.sh
+
+  namespaced-integration-test:
+    runs-on: ubuntu-latest
+    needs: [build]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/download-artifact@v2
+        with:
+          name: images
+      - name: Load Docker images
+        run: |
+          docker load -i ${GITHUB_SHA}_image.tar
+          docker load -i ${GITHUB_SHA}_hook.tar
+      - name: Install yq and bash
+        run: |
+          sudo snap install yq
+          sudo apt update
+          sudo apt install bash -y
+      - name: Create KinD cluster
+        run: |
+          GO111MODULE="on" go get sigs.k8s.io/kind
+          kind create cluster --wait 120s
+      - name: Check KinD cluster
+        run: kubectl get nodes
+      - name: Add images to KinD
+        run: |
+          kind load docker-image $(yq e '.deployment.image' helm/values.yaml)
+          kind load docker-image $(yq e '.deployment.helmHookImage' helm/values.yaml)
+
+      - name: Run actual integration test
+        run: bash connaisseur/tests/integration/namespaced-integration-test.sh
```

Comment:
can we avoid the code duplication here for github actions?

Reply:
There's [a workflow trigger thingy for Actions](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#workflow_run), but that does not allow to let Workflow A spawn B as a child, but rather if A completes, B can be triggered. In practice I would assume that this leads to the release flow showing a checkmark once it triggered the integration tests and then not showing X even though they failed later-on, which I'd say is really bad for visibility of the results

Reply:
I see, then let's make an issue about this and handle it when we got the time and think it is a good thing to pick up